src/renderers/shaders/ShaderChunk/lights_lambert_vertex.glsl.js

```diff
@@ -13,8 +13,15 @@ backGeometry.viewDir = geometry.viewDir;
 
 vLightFront = vec3( 0.0 );
 
+#if defined (USE_SHADOWMAP) && NUM_HEMI_LIGHTS > 0
```

Comment:
There is no need to condition on the presence of shadows. Keep it simple:
```glsl
vLightFront = vec3( 0.0 );
vIndirectFront = vec3( 0.0 );

#ifdef DOUBLE_SIDED
	vLightBack = vec3( 0.0 );
	vIndirectBack = vec3( 0.0 );
#endif
```

Reply:
I like calling the light indirect rather than ambient.
As for the conditions, it's to prevent most uses of Lambert from doing the extra varying color, and also to prevent adding the light colors in the fragment shader. You're sure the simplicity trumps the small performance difference? I'm keen to agree, but just want to make sure.

Reply:
We can optimize for performance later if there is a performance issue. I doubt there will be.